File: internal/content/testdata/bench/go/sequence-double-inverse.go
func sequence_equation(p []int) []int {
	n := len(p)
	inv := make([]int, n+1)
	for i := 0; i < n; i++ {
		inv[p[i]] = i + 1
	}
	result := make([]int, n)
	for x := 1; x <= n; x++ {
		result[x-1] = inv[inv[x]]
	}
	return result
}

File: internal/content/testdata/bench/csharp/sequence-double-inverse.cs
public class Solution {
    public long[] sequence_equation(long[] p) {
        int n = p.Length;
        long[] inv = new long[n + 1];
        for (int i = 0; i < n; i++) {
            inv[p[i]] = i + 1;
        }
        long[] result = new long[n];
        for (int x = 1; x <= n; x++) {
            result[x - 1] = inv[inv[x]];
        }
        return result;
    }
}
